src/main/java/hunternif/mc/atlas/RegistrarAntiqueAtlas.java

```diff
@@ -0,0 +1,65 @@
+package hunternif.mc.atlas;
+
+import hunternif.mc.atlas.item.ItemAtlas;
+import hunternif.mc.atlas.item.ItemEmptyAtlas;
+import hunternif.mc.atlas.item.RecipeAtlasCloning;
+import hunternif.mc.atlas.item.RecipeAtlasCombining;
+import net.minecraft.client.renderer.block.model.ModelResourceLocation;
+import net.minecraft.init.Items;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.crafting.IRecipe;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.client.event.ModelRegistryEvent;
+import net.minecraftforge.client.model.ModelLoader;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+import net.minecraftforge.oredict.ShapelessOreRecipe;
+
+@Mod.EventBusSubscriber(modid = AntiqueAtlasMod.ID)
+@GameRegistry.ObjectHolder(AntiqueAtlasMod.ID)
+public class RegistrarAntiqueAtlas {
+
+    @GameRegistry.ObjectHolder("empty_antique_atlas")
+    public static final ItemEmptyAtlas EMPTY_ATLAS = new ItemEmptyAtlas();
```

Comment:
This is wrong... They should be null (not initialized) so that forge will inject.